Translation/en/RedirectedResources/assets/abdata/communication/info_70/communication_40/translation.txt

```diff
@@ -1,28 +1,28 @@
-//「貴殿を見ていると、何故か戦意が高揚したようになる……不思議な感覚だ」=
-//「その、なんだ……我は女としてどうだ？　貴殿の率直な意見を聞かせて頂きたい」=
-//「んなっ！？　我の死角に入っただと！　貴殿……なかなかやるな」=
-//「我に何か用か？　見ての通りの武骨者ゆえ、役に立てることは少ないと思うぞ？」=
-//「貴殿は変わり者だな。我のような得体の知れない者は、普通避けるだろう」=
-//「なぁ……我がもし、この世界の……いや、何でもない。忘れてくれ」=
-//「もしかして……ファンタジー小説とかが好きなのかな？」=
-//「そうだな、そういうことにしておいてくれ。その方が何かと都合がいい」=
-//「はぁ、またその類の質問か……答えは否だ。まぁ、奇妙な繋がりは感じるが……」=
-//「実は凄い貴族のお嬢様だったりする？」=
-//「はは、凄いとまでは言えないが……まぁ、食うに困るようなことはなかったな」=
-//「まさか。それなりに裕福な暮らしはさせてもらっていたが、貴族なんて柄ではない」=
-//「貴殿は遠慮という言葉を覚えた方が良い。自重でも構わんぞ」=
-//「[H名]って、剣道とか強そうだよね」=
+「なっ、何を考えている！　この変態め！　即刻出ていけ！」=“Y, you pervert! What do you think you're doing?　Get out of here now!”
+「ほう……勇気と蛮勇は違うということ、その体にたっぷり教え込んでやろうか？」=“Hoo, How about I give you a good lesson on the difference between courage and bravery?”
+「すまない。今は貴殿に構っている余裕が無くてな……」=“I'm sorry. I don't have time to keep you company right now......”
+「ん……来るがいい！　我が剣の錆にしてくれる……！　すぅ……んぅぅ……」=“Hmm...... come on!　You will become rust in my sword......!　Suuuu...... Hmm......”
+「くぅ……ジロジロ見るんじゃない！　し、視線がいやらしいぞっ！」=“Kuu…… Stop staring at me!　Your gaze is too obscene!”
+「むぅ、いくら水中戦を想定しているとはいえ、この装備は少々心許ないな……」=“Muu, even assuming it is designed for underwater warfare, I feel anxious with this equipment......”
+「ここが目的地か？　それとも何か不穏な気配でも感じたか？」=“Is this where we're heading?　Or have you sensed something disturbing?”
+「待て、一つ忠告だ……我は貴殿を信頼して付いてきた。それを無下にしてくれるなよ？」=“Wait, a word of advice:...... I'm following you because I trust you. Don't disregard that, okay?”
+「貴殿を見ていると、何故か戦意が高揚したようになる……不思議な感覚だ」=“For some reason, when I look at you, I feel my fighting spirit rising...... It is a odd feeling.”
+「その、なんだ……我は女としてどうだ？　貴殿の率直な意見を聞かせて頂きたい」=“Uh, what...... do you think of me as a woman? I would like to hear your honest opinion.”
+「んなっ！？　我の死角に入っただと！　貴殿……なかなかやるな」=“No way!?　You entered my blind spot!　You ...... are pretty good.”
+「我に何か用か？　見ての通りの武骨者ゆえ、役に立てることは少ないと思うぞ？」=“What do you want with me?　As you can see, I'm a woman of strong character, so I'm of little use to you, don't you think?”
+「貴殿は変わり者だな。我のような得体の知れない者は、普通避けるだろう」=“You are a odd one. Most people would avoid a stranger like me.”
+「なぁ……我がもし、この世界の……いや、何でもない。忘れてくれ」=“Hey ...... if I am the world's...... no, it's nothing. Forget it.”
+「もしかして……ファンタジー小説とかが好きなのかな？」=“Maybe you like ...... stuff like fantasy novels?”
+「そうだな、そういうことにしておいてくれ。その方が何かと都合がいい」=“Right, Let's just leave it at that. It's more convenient this way.”
+「はぁ、またその類の質問か……答えは否だ。まぁ、奇妙な繋がりは感じるが……」=“Huh, another one of those questions ...... The answer is no. Well, I do feel a strange connection .......”
+「実は凄い貴族のお嬢様だったりする？」=“Are you actually a great noblewoman?”
+「はは、凄いとまでは言えないが……まぁ、食うに困るようなことはなかったな」=“Haha, I wouldn't say great, but ...... well, I naver had any trouble eating.”
+「まさか。それなりに裕福な暮らしはさせてもらっていたが、貴族なんて柄ではない」=“Indeed. I've been allowed to live a reasonably well-off life, but I'm not a aristocrat.”
+「貴殿は遠慮という言葉を覚えた方が良い。自重でも構わんぞ」=“You should learn the word "restraint". And learn "prudence" as well.”
+「[H名]って、剣道とか強そうだよね」=“[H名], seems like you would be strong in sports like kendo.”
```

Comment:
It's enough to use 3 dots for `……`
```suggestion
「なっ、何を考えている！　この変態め！　即刻出ていけ！」=“Y, you pervert! What do you think you're doing?　Get out of here now!”
「ほう……勇気と蛮勇は違うということ、その体にたっぷり教え込んでやろうか？」=“Hoo, How about I give you a good lesson on the difference between courage and bravery?”
「すまない。今は貴殿に構っている余裕が無くてな……」=“I'm sorry. I don't have time to keep you company right now...”
「ん……来るがいい！　我が剣の錆にしてくれる……！　すぅ……んぅぅ……」=“Hmm... come on!　You will become rust in my sword...!　Suuuu... Hmm...”
「くぅ……ジロジロ見るんじゃない！　し、視線がいやらしいぞっ！」=“Kuu…… Stop staring at me!　Your gaze is too obscene!”
「むぅ、いくら水中戦を想定しているとはいえ、この装備は少々心許ないな……」=“Muu, even assuming it is designed for underwater warfare, I feel anxious with this equipment...”
「ここが目的地か？　それとも何か不穏な気配でも感じたか？」=“Is this where we're heading?　Or have you sensed something disturbing?”
「待て、一つ忠告だ……我は貴殿を信頼して付いてきた。それを無下にしてくれるなよ？」=“Wait, a word of advice... I'm following you because I trust you. Don't disregard that, okay?”
「貴殿を見ていると、何故か戦意が高揚したようになる……不思議な感覚だ」=“For some reason, when I look at you, I feel my fighting spirit rising... It is a odd feeling.”
「その、なんだ……我は女としてどうだ？　貴殿の率直な意見を聞かせて頂きたい」=“Uh, what... do you think of me as a woman? I would like to hear your honest opinion.”
「んなっ！？　我の死角に入っただと！　貴殿……なかなかやるな」=“No way!?　You entered my blind spot!　You... are pretty good.”
「我に何か用か？　見ての通りの武骨者ゆえ、役に立てることは少ないと思うぞ？」=“What do you want with me?　As you can see, I'm a woman of strong character, so I'm of little use to you, don't you think?”
「貴殿は変わり者だな。我のような得体の知れない者は、普通避けるだろう」=“You are a odd one. Most people would avoid a stranger like me.”
「なぁ……我がもし、この世界の……いや、何でもない。忘れてくれ」=“Hey... if I am the world's... no, it's nothing. Forget it.”
「もしかして……ファンタジー小説とかが好きなのかな？」=“Maybe you like... stuff like fantasy novels?”
「そうだな、そういうことにしておいてくれ。その方が何かと都合がいい」=“Right, Let's just leave it at that. It's more convenient this way.”
「はぁ、またその類の質問か……答えは否だ。まぁ、奇妙な繋がりは感じるが……」=“Huh, another one of those questions... The answer is no. Well, I do feel a strange connection...”
「実は凄い貴族のお嬢様だったりする？」=“Are you actually a great noblewoman?”
「はは、凄いとまでは言えないが……まぁ、食うに困るようなことはなかったな」=“Haha, I wouldn't say great, but... well, I naver had any trouble eating.”
「まさか。それなりに裕福な暮らしはさせてもらっていたが、貴族なんて柄ではない」=“Indeed. I've been allowed to live a reasonably well-off life, but I'm not a aristocrat.”
「貴殿は遠慮という言葉を覚えた方が良い。自重でも構わんぞ」=“You should learn the word "restraint". And learn "prudence" as well.”
「[H名]って、剣道とか強そうだよね」=“[H名], seems like you would be strong in sports like kendo.”
```